open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -220,58 +219,45 @@ In addition, the Authorization Server
 12. shall support refresh tokens
 13. shall issue access tokens with an expiry no greater than 900 seconds and no less than 300 seconds
 
-#### ID Token as detached signature
+#### Token de ID como assinatura separada
 
-The Authorization Server shall support the provisions specified in clause 5.2.2.1 of [Financial-grade API Security Profile 1.0 - Part 2: Advanced][FAPI-1-Advanced]
+O Servidor de Autorização deve suportar as disposições especificadas na cláusula 5.2.2.1 de [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced]
 
-In addition, if the `response_type` value `code id_token` is used, the Authorization Server
+Além disso, se o valor `response_type` `code id_token` for usado, o servidor de autorização:
 
-1. should not return sensitive PII in the ID Token in the authorization response, but if it needs to,
-then it shall encrypt the ID Token.
+1. não deve retornar Informação de Identificação Pessoal (PII) confidenciais no token de ID na resposta de autorização, mas se for necessário,
+então ele deve criptografar o token de ID.
 
-#### Requesting the "cpf" Claim
+#### Solicitando a reivindicação "cpf"
```

Comment:
As Security WG had decide do not use OIDC claims to inform cpf/cnpj to AS, I suggest remove this section. At the end of this doc we have now to explain how does it going to work using atributs passed throught consent´s API

Reply:
the communication from the security working group was to continue to define CPF and CNPJ as the ecosystem consistent OIDC claims to facilitate data ownership correlation between banks and as such both the claims and mechanisms for support them are required.

Removing this section would be a normative change and will need to go into Implementers Draft 2 as Banks and TPPs have been told to expect these claims and the conformance harness will require their presence in order to pass.